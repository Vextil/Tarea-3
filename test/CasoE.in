CrearBinario mueslera 1 1 pereira 16 gimenez 2 godin 3 ppereira 6 sanchez 19 arevalo 17 lodeiro 14 rodriguez 7 rolan 9 cavani 21 FIN
ImprimirBinario 1
BuscarABB 1 gimenez
FiltroBinario 1 7 > 2 
ImprimirBinario 2
Encolar 1
Encolar 2
Desencolar
Linealizar 1 1
ImprimirLista 1
RemoverDeLista 1 3
CrearBalanceado 1 3
Izquierdo 3 4
Derecho 3 5
AlturaCantidadBinario 4 
AlturaCantidadBinario 5
AgregarProcesos notepad 100 tomcat 10 jetty 20 gvsig 20 ionic 30 geoserver 40 qgis 50 gvsig 60 firefox 70 chrome 80 gimp 90 FIN 
ListarProcesos
MuyConsumidores 40
EliminarProceso gvsig
ImprimirProcesos
Fin
