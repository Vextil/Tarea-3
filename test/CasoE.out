1>OK
2>
    (19,sanchez)
      (9,rolan)
     (7,rodriguez)
   (6,ppereira)
  (16,pereira)
 (1,mueslera)
    (14,lodeiro)
   (3,godin)
  (2,gimenez)
    (21,cavani)
   (17,arevalo)
3>
   (14,lodeiro)
  (3,godin)
 (2,gimenez)
   (21,cavani)
  (17,arevalo)
4>OK
5>
  (6,ppereira)
 (1,mueslera)
   (3,godin)
  (2,gimenez)
6>OK
7>OK
8>
    (19,sanchez)
      (9,rolan)
     (7,rodriguez)
   (6,ppereira)
  (16,pereira)
 (1,mueslera)
    (14,lodeiro)
   (3,godin)
  (2,gimenez)
    (21,cavani)
   (17,arevalo)

9>OK
10>Ordenada. arevalo cavani gimenez godin lodeiro mueslera pereira ppereira rodriguez rolan sanchez 
11>gimenez
12>OK
13>OK
14>OK
15>Altura: 3; Cantidad: 5
16>Altura: 3; Cantidad: 4
17>Hay 11 procesos.
18>boot chrome firefox geoserver gimp gvsig ionic jetty notepad qgis tomcat 
19>boot chrome firefox gimp notepad qgis 
20>La memoria que se libera es 20.
21>
(80,chrome) 
(70,firefox) 
(40,geoserver) (30,ionic) 
(90,gimp) (50,qgis) 
(20,jetty) (10,tomcat) 
(100,notepad) 
(100,boot) 
22>Fin
